### ISSUE TITLE
Beta 5.5 crashes when file open or export a file

### ISSUE THREAD

**Issue by schuessm** [Ultimaker/Cura#17033]
### Cura Version

5.5.0 Beta1

### Operating System

Ubuntu Linux 22.04.2

### Printer

Own Printer with Klipper

### Reproduction steps

1. When opening a File it Dumps CURA
2. Restart, Open file Dump Again
3. Export Settings dump/close Cura
4. 

### Actual results

When I access to the linux Filesystem with opening/Saving or Export profiles it always crashes/closing Cura

### Expected results

I can Save without a chrash

### Add your .zip and screenshots here ⬇️

[cura.zip](https://github.com/Ultimaker/Cura/files/13045612/cura.zip)

**Comment by bullestock** [comment]
Confirmed on Arch, using UltiMaker-Cura-5.5.0-beta.1-linux-X64.AppImage. I get this in the console:

```
PyInstaller/loader/pyimod02_importers.py:352: DeprecationWarning: the imp module is deprecated in favour of importlib and slated for removal in Python 3.12; see the module's documentation for alternative uses
Gtk-Message: 23:59:30.588: Failed to load module "xapp-gtk3-module"
Gtk-Message: 23:59:30.592: Failed to load module "xapp-gtk3-module"
Gtk-Message: 23:59:30.596: Failed to load module "canberra-gtk-module"
Gtk-Message: 23:59:30.600: Failed to load module "canberra-gtk-module"
Cyclic dependency detected between "file:///tmp/.mount_UltiMaJSsAqN/share/cura/resources/qml/Actions.qml" and "file:///tmp/.mount_UltiMaJSsAqN/share/cura/resources/qml/Actions.qml"

(UltiMaker-Cura:11244): Gtk-WARNING **: 23:59:58.683: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/elementary/status/16/image-missing.svg: Unable to load image-loading module: libpixbufloader-svg.so: libpixbufloader-svg.so: cannot open shared object file: No such file or directory (gdk-pixbuf-error-quark, 5)
Bail out! Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/elementary/status/16/image-missing.svg: Unable to load image-loading module: libpixbufloader-svg.so: libpixbufloader-svg.so: cannot open shared object file: No such file or directory (gdk-pixbuf-error-quark, 5)
Aborted (core dumped)
```
[stderr.log](https://github.com/Ultimaker/Cura/files/13048083/stderr.log)
[cura.log](https://github.com/Ultimaker/Cura/files/13048084/cura.log)

**Comment by jellespijker** [comment]
duplicate of https://github.com/Ultimaker/Cura/issues/16932

Will close this ticket in favor of the other one. This should also be fixed in 5.5 stable can you try it out and give your feedback in https://github.com/Ultimaker/Cura/issues/16932
